assets/src/blocks/TakeActionBoxout/TakeActionBoxoutEditor.js

```diff
@@ -20,12 +20,12 @@ const {__} = wp.i18n;
 // Planet 4 settings (Planet 4 >> Defaults content >> Take Action Covers default button text).
 const DEFAULT_BUTTON_TEXT = window.p4bk_vars.take_action_covers_button_text || __('Take action', 'planet4-blocks');
 
-const {
+import {
```

Comment:
This should be moved to earlier in the file, to be together with the other `import` statements